components/Banner/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { useAuthContext, useModalContext, MODAL_TYPES } from '../Provider';
+import { Background } from './Banner.styled';
+import { formatPrice } from '../../utils';
+
+const Banner = (): JSX.Element => {
+  const { currentUserBalance } = useAuthContext();
+  const router = useRouter();
+  const { openModal } = useModalContext();
+  const [isBannerVisible, setIsBannerVisible] = useState<boolean>(false);
+  const path = router.pathname.split('/')[1];
+
+  useEffect(() => {
+    if (!currentUserBalance) return;
+
+    const balance = parseInt(currentUserBalance.match(/\d/g)[0]);
```

Comment:
We need to bring back the `proton.getAtomicMarketBalance` that was deleted in this [commit](https://github.com/ProtonProtocol/nft-demo/commit/980ed0196613c4234029a831ca9f408e0a300360#diff-cb4cd8c9c22d04266b99214df0fe5225a7e4deec6d7aeeaeecc067dfbf221017). We should should this balance instead of the `currentUserBalance` because the `currentUserBalance` was updated to pull from the Proton wallet balance instead of the Atomic Market balance.

Reply:
Let's also stay consistent and use `parseFloat` instead of `parseInt` and use `.replace(',', '')` instead of `.match(...)` to easier readability.